source/install/install-rhel-66-postgresql.rst

```diff
@@ -0,0 +1,95 @@
+..  _install-rhel-66-postgresql:
+
+Installing PostgreSQL Database
+==============================
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Download the PostgreSQL 9.4 Yum repository.
+
+  ``wget https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-3.noarch.rpm``
+
+3. Install the Yum repository from the file that you downloaded.
+
+  ``sudo yum localinstall pgdg-redhat94-9.4-1.noarch.rpm``
+
+4. Install PostgreSQL.
+
+  ``sudo yum install postgresql94-server postgresql94-contrib``
+
+5. Initialize the database.
+
+  ``sudo service postgresql-9.4 initdb``
+
+6. Set PostgreSQL to start on boot.
+
+  ``sudo chkconfig postgresql-9.4 on``
+
+7. Start the PostgreSQL server.
+
+  ``sudo service postgresql-9.4 start``
+
+8. Switch to the *postgres* Linux user account that was created during the installation.
+
+  ``sudo --login --user postgres``
```

Comment:
The commands are equivalent, it's just that I thought the long form is more descriptive of what's going on.